projects/components/src/filter-button/in-filter-button.component.ts

```diff
@@ -0,0 +1,97 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnChanges, Output } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { TypedSimpleChanges } from '@hypertrace/common';
+import { FilterAttribute } from '../filter-bar/filter-attribute';
+import { UserFilterOperator } from '../filter-bar/filter/filter-api';
+import { IconSize } from '../icon/icon-size';
+import { FilterButtonService } from './filter-button.service';
+
+@Component({
+  selector: 'htc-in-filter-button',
+  styleUrls: ['./in-filter-button.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="filter-button" *ngIf="this.isSupported">
+      <htc-popover [closeOnClick]="false" (popoverOpen)="this.onPopoverOpen()" (popoverClose)="this.onPopoverClose()">
```

Comment:
can we use the multi-select component here?

Reply:
I can give it a shot and see.

Reply:
Ah, just looked at what that component is. The multiselect dropdown isn't quite what we want here. Just simple checkboxes.

Reply:
Can you attach a screenshot / recording? I'm getting the impression that the only difference with the multi-select is that a label isn't used. If that is the case, we can make it optional and reduce the redundancy.

Reply:
Can you attach a screenshot / recording? I'm getting the impression that the only difference with the multi-select is that a label isn't used. If that is the case, we can make it optional and reduce the redundancy.

Reply:
I think this one is unresolved yet. Jake can you please check?

Reply:
Thanks. This was buried under a "Viewed" file so I missed it. Sorry about that.

There are a few incompatibilities with the current multiselect. Yes, the label is one but also the long term vision for this dropdown is more than just checkboxes. We may add additional column actions like removing it to the same dropdown.

I've attached the mock for the current usage, but this will grow.

<img width="143" alt="Screen Shot 2020-08-26 at 1 54 42 PM" src="https://user-images.githubusercontent.com/45181984/91356106-f7205c00-e7a3-11ea-9a77-60e2a9271eba.png">

Reply:
gotcha.